core/geotime/src/main/java/mil/nga/giat/geowave/core/geotime/index/dimension/TemporalBinningStrategy.java

```diff
@@ -211,6 +237,17 @@ public int getFixedBinIdSize() {
 				return StringUtils.stringToBinary((Integer.toString(value.get(Calendar.YEAR)) + "_"
 						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.MONTH)) + "_" + TWO_DIGIT_NUMBER.format(value
 						.get(Calendar.DAY_OF_MONTH))));
+			case HOUR:
+				return StringUtils.stringToBinary((Integer.toString(value.get(Calendar.YEAR)) + "_"
+						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.MONTH)) + "_"
+						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.DAY_OF_MONTH)) + "_" + TWO_DIGIT_NUMBER
+						.format(value.get(Calendar.HOUR_OF_DAY))));
+			case MINUTE:
+				return StringUtils.stringToBinary((Integer.toString(value.get(Calendar.YEAR)) + "_"
+						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.MONTH)) + "_"
+						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.DAY_OF_MONTH)) + "_"
+						+ TWO_DIGIT_NUMBER.format(value.get(Calendar.HOUR_OF_DAY)) + "_" + TWO_DIGIT_NUMBER
+						.format(value.get(Calendar.MINUTE))));
```

Comment:
I'd suggest you document an approach in an issue but don't worry about implementing it yet - tag the issue as "breaking changes"